storage/azure/src/index.ts

```diff
@@ -2,7 +2,7 @@
  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
-export * from "./client";
 export * from "./common";
-export * from "./frontend";
+export * from "./client";
 export * from "./server";
+export * from "./frontend";
```

Comment:
Done, anything that runs on node.js imports the package itself, frontend imports @package/lib/frontend, common - @package/lib/common since frontend depends on it

Reply:
done, frontend index now imports common